src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Application {
+    public static void main(String[] args) {
+        int figureNumber = 11;

```

Comment:
The variable 'figureNumber' could be renamed to a more descriptive name like 'totalNumberOfFigures' for better code readability.